templates/isdialogmote/innkalling-arbeidsgiver.hbs

```diff
@@ -58,10 +69,27 @@
     <title>Dialogmøte</title>
 </head>
 <body>
-<h1>Innkalling til dialogmøte</h1>
-
-<h2>Tid og sted</h2>
-<p>{{ innkalling.tidOgSted.sted }}</p>
-
+<h1>Til nærmeste leder</h1>
+{{#each .}}
+    {{#eq type 'HEADER'}}
+        <h2>{{text}}</h2>
+    {{/eq}}
+    {{#eq type 'PARAGRAPH'}}
+        {{#if title}}
+            <h3>{{title}}</h3>
+        {{/if}}
+        {{#if text }}
+            <p>{{text}}</p>
+        {{/if}}
+    {{/eq}}
```

Comment:
Man skal vel kun få én component her? Da kan det kanskje gå an å flytte `if title -> <p>title</p>` øverst, i stedet for å sjekke inne i hver typesjekk? Det skal også gå an å gjøre `{{else eq type 'TYPE'}}`, så sparer man noen linjer ved å slippe å lukke/åpne `eq` flere ganger.

Reply:
Fikset.